weather_forecast/3dcorrection/unets/configs/unet.yaml

```diff
@@ -0,0 +1,35 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+model:
+    class_path: models.LitUnet1D
+    init_args:
+        data_path: data/
```

Comment:
It shouldn't be /data/processed ?

Reply:
This isn't forced as the dataproc preprocessed the data on-the-fly, not storing them in a raw format. But for consistency sake, let's call it data/processed :)